applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp

```diff
@@ -19,6 +19,7 @@
 #include "containers/model.h"
 #include "includes/checks.h"
 #include "includes/model_part.h"
+#include "includes/parallel_environment.h"
```

Comment:
```suggestion
```

Reply:
This is required in order to call the factory of the `ParallelFillCommunicator`.

Reply:
ups :)